components/tidb_query_datatype/src/codec/mysql/json/comparison.rs

```diff
@@ -258,7 +258,7 @@ mod tests {
             let left: Json = left_str.parse().unwrap();
             let right: Json = right_str.parse().unwrap();
             assert!(left < right);
-            assert_eq!(left, left);
+            // assert_eq!(left, left);
```

Comment:
Remove?

Reply:
I'm not sure if the test writer intends to test the equal case, is it necessary to use `assert_eq!(left, left.clone())` instead?

Reply:
Oh, kinda makes sense. I think clone sounds good, or we can just suppress the warning.